js/gated-content/src/components/MediaPlayer.vue

```diff
@@ -53,6 +53,14 @@ export default {
       this.playbackLogged = true;
       this.handlePlayerEvent('videoPlaybackStarted');
     },
+    videoId() {
+      let embedObj = this.media.field_media_video_embed_field;
+      if (this.media.field_media_source === 'youtube_playlist') {
+        embedObj = embedObj.match(/(\?|&)v=([^&#]+)/).pop();
+        this.media.field_media_video_id = embedObj;
```

Comment:
Instead of overriding the origin object let's just return the value.
Please use
```js
return embedObj;
```
instead of 
```js
this.media.field_media_video_id = embedObj;
```